src/Illuminate/Console/Scheduling/ScheduleGroup.php

```diff
@@ -0,0 +1,221 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Illuminate\Console\Scheduling;
+
+use Closure;
+use DateTimeZone;
+
+class ScheduleGroup
```

Comment:
Class is missing doc blocks on some methods, which must be implemented on all methods (including constructor).

Reply:
I have added the missing doc blocks. Thanks 👍